tests/test_jump_cas22.py

```diff
@@ -485,51 +480,37 @@ def test_find_jumps(jump_data):
             assert fit["index"][0]["start"] == 0
             assert fit["index"][0]["end"] == len(read_pattern) - 1
         else:
-            # There should be a single jump detected; however, this results in
-            # two resultants being excluded.
-            if resultant_index not in fit["jumps"]:
-                incorrect_does_not_capture += 1
-                continue
-            if len(fit["jumps"]) < 2:
-                incorrect_too_few += 1
-                continue
-            if len(fit["jumps"]) > 2:
-                incorrect_too_many += 1
-                continue
-
-            # The two resultants excluded should be adjacent
-            jump_correct = [
-                (jump in (resultant_index, resultant_index - 1, resultant_index + 1)) for jump in fit["jumps"]
-            ]
-            if not all(jump_correct):
-                incorrect_other += 1
-                continue
-
+            # Check that the inserted jump is detected or if the jump occurs in the last resultant
+            #    (there are some unresolved issues with this case)
+            assert resultant_index in fit["jumps"] or resultant_index == resultants.shape[0] - 1
```

Comment:
This points out the one exception. It appears that it does not always detect jumps (like it should) if they occur in the final resultant. This should be investigated more.